src/pages/SubmissionDetails.test.js

```diff
@@ -0,0 +1,174 @@
+        workflow_url: 'https://abc.wdl',
+        last_modified_timestamp: '2022-01-28T22:30:05.291Z',
+        workflow_params: '[{"parameter_name":"workflow_input_foo","parameter_type":"String","source":{"type":"literal","entity_attribute":"helloworld"}},{"parameter_name":"workflow_input_foo_rating","parameter_type":"Int","source":{"type":"entity_lookup","entity_attribute":"entity_field_foo_rating"}}]'
+      },
+      {
+        run_id: 'b7234aae-6f43-405e-bb3a-71f924e09825',
+        state: 'Failed',
+        submission_date: '2022-07-14T22:22:15.591Z',
+        workflow_url: 'https://xyz.wdl',
+        last_modified_timestamp: '2022-07-14T23:14:25.791Z',
+        workflow_params: '[{"parameter_name":"workflow_input_foo","parameter_type":"String","source":{"type":"literal","entity_attribute":"helloworld"}},{"parameter_name":"workflow_input_foo_rating","parameter_type":"Int","source":{"type":"entity_lookup","entity_attribute":"entity_field_foo_rating"}}]'
+      }
+    ]
+  }
+
+  beforeAll(() => {
+    Object.defineProperty(HTMLElement.prototype, 'offsetHeight', { configurable: true, value: 1000 })
+    Object.defineProperty(HTMLElement.prototype, 'offsetWidth', { configurable: true, value: 800 })
+  })
+
+  beforeEach(() => {
+    const getRunsMethod = jest.fn(() => Promise.resolve(runsData))
+    Ajax.mockImplementation(() => {
+      return {
+        Cbas: {
+          runs: {
+            get: getRunsMethod
+          }
+        }
+      }
+    })
+  })
+
+  afterEach(() => {
+    jest.clearAllMocks()
+  })
+
+  afterAll(() => {
+    Object.defineProperty(HTMLElement.prototype, 'offsetHeight', originalOffsetHeight)
+    Object.defineProperty(HTMLElement.prototype, 'offsetWidth', originalOffsetWidth)
+  })
+
+  it('should correctly display previous 2 runs', async () => {
+    // Act
+    await act(async () => {
+      await render(h(SubmissionDetails))
+    })
+
+    const table = screen.getByRole('table')
+
+    // Assert
+    expect(table).toHaveAttribute('aria-colcount', '6')
+    expect(table).toHaveAttribute('aria-rowcount', '3')
+
+    const rows = within(table).queryAllByRole('row')
+    expect(rows.length).toBe(3)
+
+    const headers = within(rows[0]).queryAllByRole('columnheader')
+    expect(headers.length).toBe(6)
+    within(headers[0]).getByText('Run ID')
+    within(headers[1]).getByText('Status')
+    within(headers[2]).getByText('Submission date')
+    within(headers[3]).getByText('Duration')
+    within(headers[4]).getByText('Data')
+    within(headers[5]).getByText('Logs')
+
+    // check data rows are rendered as expected
+    const cellsFromDataRow1 = within(rows[1]).queryAllByRole('cell')
+    expect(cellsFromDataRow1.length).toBe(6)
+    within(cellsFromDataRow1[0]).getByText('b7234aae-6f43-405e-bb3a-71f924e09825')
+    within(cellsFromDataRow1[1]).getByText('Failed')
+    within(cellsFromDataRow1[2]).getByText(/Jul 14, 2022/)
+    // TODO: Add 'within for 'duration' once WM-1461 is complete
```

Comment:
I think the check for duration can be added now as it only relies on mock data and not the actual API response